crates/nu-command/src/env/with_env.rs

```diff
@@ -63,8 +63,8 @@ impl Command for WithEnv {
                 result: Some(Value::test_string("Z")),
             },
             Example {
-                description: "Set by row(e.g. `open x.json` or `from json`)",
-                example: r#"'{"X":"Y","W":"Z"}'|from json|with-env $in { [$env.X $env.W] }"#,
+                description: "Set by key-value record",
+                example: r#"with-env {X: "Y", W: "Z"} { [$env.X $env.W] }"#,
                 result: None,
```

Comment:
To run the documentation tests you can change that to something like `Some(Value::list(vec![Value::test_string...], Span::test_data()))

Reply:
oh yep, no problem, i'll add that asap :wink:

Reply:
Awesome 🎉

Reply:
opened in #8148 :yum: